tests/e2e/vc_reboot_volume_lifecycle.go

```diff
@@ -185,5 +194,12 @@ var _ bool = ginkgo.Describe("Verify volume life_cycle operations works fine aft
 			time.Sleep(waitTimeForCNSNodeVMAttachmentReconciler)
 			verifyCRDInSupervisor(ctx, f, pod.Spec.NodeName+"-"+svcPVCName, crdCNSNodeVMAttachment, crdVersion, crdGroup, false)
 		}
+		if vanillaCluster {
```

Comment:
May be we can use `switch` statement instead of `if...else` blocks.
It is ok either way, feel free to drop this comment.